### PR TITLE
Adding social icons (pre-embed pr)

### PR DESCRIPTION
**Pull request by ldecicco-USGS**
Took ~ 1 hour (rounding up....I'll probably round down on the next PR to compensate). I want to get this merged and finished before *really* tackling the embed stuff. Here's what it looks like:

image

### CODE REVIEW

File: viz.yaml [hunk 1]
```diff
@@ -265,11 +271,14 @@ publish:
     template: fullpage
     publisher: page
     depends:
+      sourcesansprofont: sourcesansprofont
```
Comment (ldecicco-USGS) [inline viz.yaml]: I'm mucking around with this to try to figure out why the footer looks dumb again....I don't think this is it...I'll remove

File: viz.yaml [hunk 2]
```diff
@@ -79,6 +79,12 @@ parameter:
     id: wu_type_column_names
     orig_names: ["TO-Wtotl", "PT-Wtotl", "PS-Wtotl", "IR-WFrTo", "IN-Wtotl"]
     new_names: ["total", "thermoelectric", "publicsupply", "irrigation", "industrial"]
+  - 
+    id: social-links
+    github: https://github.com/USGS-VIZLAB/water-use-15
+    twitter: https://twitter.com/share?url=TODO!!!
+    facebookLink: http://www.facebook.com/sharer.php?u=TODO!!!
```
Comment (lindsaycarr) [inline viz.yaml]: let's not forget "TODO!!!" these 😄

**Comment by lindsaycarr** [comment]
So @ldecicco-USGS is this time linked to #26 stuff or separate?

**Comment by ldecicco-USGS** [comment]
This is basically pre-#26...trying to keep the PR to a reasonable length, seemed like doing this obvious work first would help. (I need to do this for embed....but if somehow the embed work completely fails because I'm confused on url hash stuff...at least we'll still have the basics)

**Pull request merged**